### ISSUE TITLE
Alternative keyboard layout dose not work in 3.8.0 Fedora 40

### ISSUE THREAD

**Issue by LILNAZ** [FreeRDP/FreeRDP#10722]
**Describe the bug**
Alternative keyboard layouts using the flag `/kbd` and `/kbd-list` dose not work fully on 3.8.0 on Fedora 40. It did work on Fedora 39 and xfreerdp version 2.11.2

**To Reproduce**
To reproduce this bug do the following:
1. Run `xfreerdp /v:127.0.0.1 /kbd:Swedish | head` which gives the following output:
    1. `[12:17:24:199] [837561:000cc7b9] [ERROR][com.freerdp.client.common.cmdline] - [freerdp_client_settings_parse_command_line_arguments_int]: Command line parsing failed at 'kbd' value 'Swedish' [-1002]`
    2. `"/kbd:Swedish"` and  `/kbd:"Swedish"` gives the same result
3. Run `xfreerdp /v:127.0.0.1 /kbd:0x0000041D | head`  which gives the same results as the named layout:
    1. `[12:21:21:089] [838078:000cc9be] [ERROR][com.freerdp.client.common.cmdline] - [freerdp_client_settings_parse_command_line_arguments_int]: Command line parsing failed at 'kbd' value '0x0000041D' [-1002]`
    2. `"/kbd:0x0000041D"` and  `/kbd:"0x0000041D"` gives the same result
4. Run `xfreerdp /kbd-list | head` witch gives the following output:
    1. `[12:18:22:664] [837689:000cc839] [ERROR][com.winpr.commandline] - [log_error]: Failed at index 1 [/kbd-list]: Invalid sigil` 

Do not know if the syntax has changed as this now work:
-  `xfreerdp /v:127.0.0.1 /kbd:layout:0x0000041D`

However, the named version did not work:
1. Run `xfreerdp /v:127.0.0.1 /kbd:layout:Swedish` witch gives the following output:
```
[12:47:16:133] [841329:000cd671] [ERROR][com.freerdp.client.common.cmdline] - [parse_kbd_layout]: Could not identify keyboard layout: Swedish
[12:47:16:133] [841329:000cd671] [ERROR][com.freerdp.client.common.cmdline] - [parse_kbd_layout]: Use /list:kbd to list available layouts
[12:47:16:133] [841329:000cd671] [ERROR][com.freerdp.client.common.cmdline] - [freerdp_client_settings_parse_command_line_arguments_int]: Command line parsing failed at 'kbd' value 'layout:Swedish' [-1002]
```

**Expected behavior**
The flags should be parsed and used to give the correct keyboard layout.

**Application details**
* FreeRDP version (`xfreerdp /version`): `FreeRDP version 3.8.0 (n/a)`
* Command line used: `/kbd` and `/kbd-list`
* OS version connecting to (server side): **N/A**
* If available the log output from a run with `/log-level:trace 2>&1 | tee log.txt`: **Gave no extra log output**

**Environment (please complete the following information):**
 - OS: Linux
 - Version/Distribution: Fedora 40
 - Architecture: [amd64, arm]: x86_64

**Comment by LILNAZ** [comment]
The list of available keyboard layouts is less in 3.8.0 (Fedora 40) then it was in 2.11.5 (Fedora 39). I cannot find Swedish in 3.8.0.

On 3.8.0 running `xfreerdp /list:kbd | grep 0x0 | wc -l` gives the result of 42 while on 2.11.5 running `xfreerdp /kbd-list | grep 0x0 | wc -l` gives 242.

**Comment by akallabeth** [comment]
duplicates https://github.com/FreeRDP/FreeRDP/issues/10636
